src/admin-views/editor/panel/ticket.php

```diff
@@ -240,7 +499,21 @@ class="ticket_field"
 					id="ticket_form_save"
 					class="button-primary tribe-dependent tribe-validation-submit"
 					name="ticket_form_save"
-					value="<?php echo esc_attr( sprintf( _x( 'Save %s', 'meta box ticket form button text', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'meta_box_ticket_form_button_text' ) ) ); ?>"
+					value="
```

Comment:
Needs one-liner treatment (define string at top, echo and escape on one line), specifically no extra whitespace in the attribute quotes.

Reply:
done